geoapps/inversion/electricals/direct_current/two_dimensions/constants.py

```diff
@@ -422,15 +408,13 @@
         "value": 0.0,
         "visible": False,
     },
-    "resolution": None,
-    "detrend_order": None,
-    "detrend_type": None,
     "tile_spatial": 1,
     "out_group": {"label": "Results group name", "value": "DirectCurrentInversion"},
 }
 
 default_ui_json = dict(base_default_ui_json, **default_ui_json)
-
+for key in ["resolution", "detrend_order", "detrend_type", "y_norm", "alpha_y"]:
```

Comment:
this might not be needed since we cut down the default_ui_json in the inversion params class:
![image](https://github.com/MiraGeoscience/geoapps/assets/81254271/2e075c96-da2e-46f2-b9f3-37db417d3eff)

Reply:
Correct, gone.